src/EnergyPlus/SolarShading.cc

```diff
@@ -738,6 +742,94 @@ namespace SolarShading {
 			ReportExtShadingSunlitFrac = false;
 		}
 
+		bool DisableSelfShadingWithinGroup = false;
+		bool DisableSelfShadingBetweenGroup = false;
+
+		if ( NumAlphas >= 6 ) {
+			if ( SameString( cAlphaArgs( 6 ), "Yes" ) ) {
+				DisableSelfShadingWithinGroup = true;
+				cAlphaArgs( 6 ) = "Yes";
```

Comment:
I don't know. Why is it assigned everywhere else in reading this object (AlphaArg 3, 4, 5)? If no particular reason, I'll remove it.

Reply:
I see why - these are being used later to report out the final status of the flags:
```
line 851 gio::write( OutputFileInits, fmtA ) << "Shadowing/Sun Position Calculations Annual Simulations," + cAlphaArgs( 1 ) + ',' + RoundSigDigits( ShadowingCalcFrequency ) + ',' + RoundSigDigits( MaxHCS ) + ',' + cAlphaArgs( 2 ) + ',' + cAlphaArgs( 3 ) + ',' + cAlphaArgs( 4 ) + ',' + cAlphaArgs( 5 );
```
Since the input from IDF could be any case, these assignments are forcing to CamelCase and resetting the string if the input was overridden.  It's unusual, but leave them here.

Sooooo, these can stay, but we should add the new inputs (args 6 and 7) to this reporting line and add their names to the header line just above.

Reply:
Added.